flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -388,10 +378,12 @@ public Task(
 			this,
 			inputGateDeploymentDescriptors,
 			metrics.getIOMetricGroup(),
-			inputGroup,
-			buffersGroup,
 			metrics.getIOMetricGroup().getNumBytesInCounter());
 
+		// we will have to check type of shuffle service later whether it is NetworkEnvironment
+		//noinspection deprecation
```

Comment:
inconsistent with other parts of the PR that use `@SuppressWarnings("deprecation")`

Reply:
it does not work for statements, at least in Idea. that is why Idea has this special extra comment, I guess.